src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -1161,175 +1161,175 @@ f1_co_directors,45,1994,Line 45,,,,
+f1_comp_balance_db,7,2004,"Nuclear Fuel in Process of Ref., Conv.,Enrich., and Fab. (120.1)",reported_value,nuclear_fuel_in_process_of_refinement_conversion_enrichment_and_fabrication,,
+f1_comp_balance_db,8,2004,Nuclear Fuel Materials and Assemblies-Stock Account (120.2),reported_value,nuclear_fuel_materials_and_assemblies_stock_account_major_only,,
+f1_comp_balance_db,9,2004,Nuclear Fuel Assemblies in Reactor (120.3),reported_value,nuclear_fuel_assemblies_in_reactor_major_only,,
+f1_comp_balance_db,10,2004,Spent Nuclear Fuel (120.4),reported_value,spent_nuclear_fuel_major_only,,
+f1_comp_balance_db,11,2004,Nuclear Fuel Under Capital Leases (120.6),reported_value,nuclear_fuel_under_capital_leases,,
+f1_comp_balance_db,12,2004,(Less) Accum. Prov. for Amort. of Nucl. Fuel Assemblies (120.5),reported_value,accumulated_provision_for_amortization_of_nuclear_fuel_assemblies,,
+f1_comp_balance_db,13,2004,Net Nuclear Fuel (Enter Total of lines 7-11 less 12),calculated_value,nuclear_fuel_net,,
+f1_comp_balance_db,14,2004,Net Utility Plant (Enter Total of lines 6 and 13),calculated_value,utility_plant_and_nuclear_fuel_net,,
+f1_comp_balance_db,15,2004,Utility Plant Adjustments (116),reported_value,other_electric_plant_adjustments,,
+f1_comp_balance_db,16,2004,Gas Stored Underground - Noncurrent (117),reported_value,gas_stored_underground_noncurrent,,
+f1_comp_balance_db,17,2004,OTHER PROPERTY AND INVESTMENTS,header,,,
+f1_comp_balance_db,18,2004,Nonutility Property (121),reported_value,nonutility_property,,
+f1_comp_balance_db,19,2004,(Less) Accum. Prov. for Depr. and Amort. (122),reported_value,accumulated_provision_for_depreciation_and_amortization_of_nonutility_property,,
+f1_comp_balance_db,20,2004,Investments in Associated Companies (123),reported_value,investment_in_associated_companies,,
+f1_comp_balance_db,21,2004,Investment in Subsidiary Companies (123.1),reported_value,investment_in_subsidiary_companies,,
+f1_comp_balance_db,22,2004,"(For Cost of Account 123.1, See Footnote Page 224, line 42)",header,,,
+f1_comp_balance_db,23,2004,Noncurrent Portion of Allowances,reported_value,noncurrent_portion_of_allowances,,This xbrl factoid is mapped to two dbf rows that might have different sign conventions.
+f1_comp_balance_db,24,2004,Other Investments (124),reported_value,other_investments,,
+f1_comp_balance_db,25,2004,Sinking Funds (125),reported_value,sinking_funds,,
+f1_comp_balance_db,26,2004,Depreciation Fund (126),reported_value,depreciation_fund,,
+f1_comp_balance_db,27,2004,Amortization Fund - Federal (127),reported_value,amortization_fund_federal,,
+f1_comp_balance_db,28,2004,Other Special Funds (128),reported_value,other_special_funds,,
+f1_comp_balance_db,29,2004,Special Funds (Non Major Only) (129),reported_value,special_funds_non_major,TRUE,"Close to special_funds, but only includes non major entities"
```

Comment:
Looking at [the XBRL taxonomy](https://xbrlview.ferc.gov/yeti/resources/yeti-gwt/Yeti.jsp#tax~(id~8*v~72)!con~(id~80270)!net~(a~143*l~35)!lang~(code~en)!path~(g~3709*p~0_0_0_0_1_10)!rg~(rg~4*p~1)) it seems like this should be mapped to `special_funds` -- they don't have the "non-major only" in the factoid, but the description is exactly the same as this row: "Special Funds (Non Major Only) (129)" -- they are often not great about following their own naming conventions. If the Label in the XBRL Taxonomy matches the `row_literal` value perfectly like this, then I think they should probably be mapped together even if the string they've used as a label doesn't have all of the detail that the label does.

Reply:
Is it weird though that `Special Funds (125-128)` and `Special Funds (Non Major Only) (129)` (what with different schedule numbers) both map to `special_funds`?

Reply:
You have to look at the XBRL taxonomy **and** the `row_literal` values:

* "Special Funds (Non Major Only) (129)" clearly maps to `special_funds`
* "Other Special Funds" (128) maps to `other_special_funds`
* "Special Funds (125-128)" doesn't exist in XBRL at all -- it looks like another case where in the early days they were lumping together several FERC Accounts (it only shows up in the earliest iteration of the rows literals, from 1994), which later get split out in the DBF data, and are currently split out in the XBRL data. So that row in 1994 should get `dbf_only = TRUE`

Reply:
Ok so I was close, I just said `dbf_only = TRUE` for `Special Funds (Non Major Only) (129)` when it should have been `Special Funds (125-128)`